support/cas-server-support-redis-core/src/main/java/org/apereo/cas/redis/core/RedisObjectFactory.java

```diff
@@ -90,7 +90,10 @@ public static RedisConnectionFactory newRedisConnectionFactory(final BaseRedisPr
                                                                    final boolean initialize,
                                                                    final CasSSLContext casSslContext) {
         var factory = (LettuceConnectionFactory) null;
-        if (redis.getSentinel() != null && StringUtils.hasText(redis.getSentinel().getMaster())) {
+        if (StringUtils.hasText(redis.getUri())) {
+            factory = new LettuceConnectionFactory(LettuceConnectionFactory.createRedisConfiguration(redis.getUri()),
```

Comment:
I think you should move this to the `getStandaloneConfig()` method and do the same check there. Less duplicate code, and it would be more consistent.

Reply:
Yes, it's cleaner. I updated both PRs.